docs/src/main/sphinx/security/cli.rst

```diff
@@ -90,17 +90,21 @@ Additional Kerberos debugging information
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can enable additional Kerberos debugging information for the Trino CLI
-process by passing ``-Dsun.security.krb5.debug=true`` as a JVM argument, when
-starting the CLI process. Doing so requires invoking the CLI JAR via ``java``
-instead of running the self-executable JAR directly. The self-executable jar
-file cannot pass the option to the JVM.
+process by passing ``-Dsun.security.krb5.debug=true``,
+``-Dtrino.client.debugKerberos=true``, and
+``-Djava.security.debug=gssloginconfig,configfile,configparser,logincontext``
+as a JVM argument, when starting the CLI process. Doing so requires invoking
+the CLI JAR via ``java`` instead of running the self-executable JAR directly.
+The self-executable jar file cannot pass the option to the JVM.
```

Comment:
JAR

Reply:
Also ... you could actually remove that whole sentence. It is redundant.

Reply:
Done!